modules/olmv1-creating-a-service-account.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/managing-ce.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="olmv1-creating-a-service-account_{context}"]
+= Creating a service account
+
+Unlike {olmv0-first}, {olmv1} does not have permissions to install, update, and manage cluster extensions. Cluster administrators must create a service account and assign the role-based access controls (RBAC) required to install, update, and manage cluster extensions.
+
+[IMPORTANT]
+====
+include::snippets/olmv1-known-issue-service-accounts.adoc[]
+====
+
+.Prerequisites
+
+* Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+
+.Procedure
+
+. Create a service account custom resource (CR), similar to the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: <extension>-installer
+  namespace: <namespace>
+----
++
+.Example `extension-service-account.yaml` file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pipelines-installer
+  namespace: pipelines
+----
+
+. Apply the service account CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f extension-service-account.yaml
+----
+
+. Create a CR to assign the RBAC to the service account:
++
+[WARNING]
+====
+The following role binding does not follow the principle of least privilege. Do not use this role binding on production clusters.
```

Comment:
You mention "role binding" here but all that is shown is a `Role` resource. I would expect both a `Role` and `RoleBinding` to be present in this example.

The `Role` defines the permissions, the `RoleBinding` "binds" them to an entity (in this case the `ServiceAccount`)

Also worth noting, using only a `Role` and `RoleBinding` will not grant sufficient permissions if the ClusterExtension includes things that need to be applied at the "cluster" level (i.e `CustomResourceDefinitions`). This would likely need to be a `ClusterRole` and `ClusterRoleBinding` pairing if the intention is to give full admin permissions (like the permissions in the example `Role` suggests).

I would recommend addressing the above two things at a minimum.

Now for a bit of an aside - I'm not sure if there is any concerns with churn on this doc in particular, but as a reader I would probably hesitate reading the provided way to install something with OLMv1 is to give unfettered access to everything on my cluster to a particular `ServiceAccount`. I would _personally_ prefer we give a more granular set of RBAC resources that would work to install all supported bundles _without_ giving access to everything. I won't block this from going in solely based on this though.

If you are interested in narrowing the set of permissions to be more explicit, I'm happy to help experiment and find the "sweet spot" of the minimal set of permissions that is still open enough to install all of the currently supported bundles.

Reply:
Unfortunately, there is going to be a lot of churn with this doc. My intention is to document the most restrictive permissions that will still work and install the supported cluster extensions in the RH catalogs.

Reply:
I fully intend to have to change and update this doc for 4.18/when the known issue referenced in this doc is backported.